src/e2e/registration.spec.js

```diff
@@ -0,0 +1,28 @@
+import { getRegistrationData } from "../utils/registration-data";
+import { UserActions } from "../project/user-actions";
+import { test } from '@playwright/test';
+
+test.describe('Check registration form', () => {
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto('https://qauto.forstudy.space/');
+    });
+
+    test('Register new customer and login', async ({ page }) => {
+        const registrationData = getRegistrationData();
+        const userActions = new UserActions(page);
```

Comment:
User Actions is not a Page Object 

It does not represent particular page

It is going to be messy